en/frames_plane/reptile_dragon_2.md

```diff
@@ -0,0 +1,391 @@
+
+The channel mappings for this build are included in the params file linked at the bottom of this page.
+I chose to map the channels in order throttle, roll, pitch, yaw, (blank), and flight mode
+
+:::note
+ExpressLRS requires AUX1 as an "arming channel".
+This arming channel is separate from PX4's arming mechanism and is used to tell the ELRS TX that is can switch into high transmit power.
+
+In the PX4 channel mappings, I simply skip over this channel.
+On my transmitter, this channel is set to always be "high", so ELRS is always armed.
+:::
+
+### Motor Setup & Prop Installation
+
+The RD2 kit comes with clockwise and counter clockwise propellers for counter rotating motors.
+With counter rotating props, the airplane can be set up such that it has no [critical motors](https://en.wikipedia.org/wiki/Critical_engine).
+
+With no critical motors, controllability will be maximized if a motor fails.
+The motor direction should be set such that props should turn towards the fuselage on top of the plane.
+In other words, if you look at the left motor with the airplane facing away from you, it should spin clockwise while the right motor should spin counter clockwise.
+
+With the propellers removed, power the airplane up and use the [Actuator](../config/actuators.md) test in QGC to spin up the motors.
+If the left or right motor does not spin in the correct direction, swap two of its ESC leads and check it again.
+Finally, when both motors are spinning the correct directions, use a wrench to attach the propellers.
+
+### Final Checks
+
+Prior to the first flight, a comprehensive preflight must be conducted.
+
+I recommend checking the following items:
+
+- Sensor calibration (QGC)
+  - Mag calibration
+  - Accelerometer calibration
+  - Airspeed calibration
+  - Level horizon calibration
+- Check control surface deflection
+ - Right stick -> Right aileron goes up, left aileron goes down
+ - Left stick -> Left aileron goes up, right aileron goes down
+ - Stick back -> elevator goes up
+ -Stick forward -> elevator goes down
+ - Left rudder -> Rudder goes left
+ - Right rudder -> Rudder goes right
+- Check Px4 inputs (in `stabilized mode`)
+ - Roll right -> Right Aileron goes down
+ - Roll left -> Left aileron goes down
+ - Pitch up -> Elevator goes down
+ - Pitch down -> Elevator goes up
+
+### First Flight
+
+I recommend performing the first takeoff in manual mode.
+Because this airplane has no landing gear, you will either need to throw the airplane yourself, or ideally have a helper throw it.
+When throwing any airplane, throw at a slightly nose up attitude with full throttle.
+
+It's critical to be ready to give aft stick input to prevent the airplane from impacting the ground if it happens to be trimmed nosedown.
+Once the airplane is successfully airborne, cruise up to an altitude of a few hundred feet and switch to [Acro mode](../flight_modes/acro_fw.md).

```

Comment:
Note, it is good to link the modes to the mode docs.

@chris1seto Is this sensible? Acro mode is harder to fly than position mode - why this order?

Reply:
I wouldn't necessarily say that acro is "harder" to fly than position mode. I've been voicing that Px4 should not discourage people from flying in manual and acro, because I think they are realistically base level skills for a user who is building their onw airplane, and without those two modes, you basically can't safely maiden a new airplane or land/takeoff. Automatic takeoff and landing needs an autonomous flight mode which needs tuning to get to. Chicken and egg problem.